packages/protocol/test/stability/grandamento.ts

```diff
@@ -767,7 +778,11 @@ contract('GrandaMento', (accounts: string[]) => {
         })
       })
 
-      it('reverts when the vetoPeriodSeconds has not elapsed since the approval time', async () => {
+      it("reverts when the proposal's vetoPeriodSeconds has not elapsed since the approval time", async () => {
+        // Set the contract's vetoPeriodSeconds to 0 to illustrate that
```

Comment:
Could add another case where the contract's veto period becomes longer and yet our veto period elapses based on the old value.